fs/tmpfs/fs_tmpfs.c

```diff
@@ -992,14 +1026,21 @@ static int tmpfs_find_file(FAR struct tmpfs_s *fs,
                            FAR struct tmpfs_directory_s **parent)
 {
   FAR struct tmpfs_object_s *to;
+  size_t len;
   int ret;
 
+  len = strlen(relpath);
+  if (len && relpath[len - 1] == '/')
```

Comment:
Ditto

Reply:
We need return fail here because it's invalid that file name contain the tail '/'.

Reply:
I mean: my point was the same:
if (len == 0)
 {
  return -EINVAL;
 }
else (relpath[len - 1] == '/') ...

Reply:
Done.